include_core/omrport.h

```diff
@@ -1463,6 +1471,10 @@ typedef struct OMRPortLibrary {
 	uint64_t ( *sysinfo_cgroup_are_subsystems_enabled)(struct OMRPortLibrary *portLibrary, uint64_t subsystemFlags);
 	/** see @ref omrsysinfo.c::omrsysinfo_cgroup_get_memlimit "omrsysinfo_cgroup_get_memlimit"*/
 	int32_t (*sysinfo_cgroup_get_memlimit)(struct OMRPortLibrary *portLibrary, uint64_t *limit);
+	/** see @ref omrsysinfo_get_container_type "omrsysinfo_get_container_type"*/
+	int32_t ( *sysinfo_get_container_type)(struct OMRPortLibrary *portLibrary, OMRContainerType *containerType);
+	/** see @ref omrsysinfo_get_container_name "omrsysinfo_get_container_name"*/
+	const char * ( *sysinfo_get_container_name)(struct OMRPortLibrary *portLibrary, OMRContainerType containerType);
```

Comment:
These two APIs don't belong in this changeset.  Given the VM only supports detecting linux cgroups or nothing, we don't have enough examples to generalize to a good portable api.  

I would rather forgo these APIs now and add them when we have second (or third) container system to work with.  That will result in a better designed api that is easier to maintain longterm.